scripts/test_upgrade/test_upgrade.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+#
+# test_upgrade.sh
+#
+# What this script does:
+# - Starts Cromwell using the "previous" cromwell JAR/configuration
+# - Sends 3 jobs to Cromwell (2 batched, one single)
+# - Waits 6 minutes
+# - Sends the same 3 jobs again, in the same way
+# - Shutdown Cromwell with jobs still running
+# - Restarts Cromwell using the "new" cromwell JAR/configuration
+# - Sends the same three jobs again
+# - Waits for everything to complete
+#
+# What it doesn't do (yet... but maybe for the C27 release!):
+# - Guarantee that any jobs are in any specific states at the point that it shuts down.
+# - Check the status of jobs after completing.
+# - Check that operations IDs weren't duplicated after restarting.
```

Comment:
Hmm these are pretty significant shortcomings.  My warmth and fuzziness may be impacted.